demo/components/Translate.tsx

```diff
@@ -0,0 +1,14 @@
+function translate(){
```

Comment:
Can you Capitalize? @nottyo

Reply:
OK. Sure.  https://github.com/nvh95/jest-preview/pull/56/commits/9bdfaae7596e70eb4f0f13555700ba92303a2806